thirdparty/CMakeLists.txt

```diff
@@ -23,6 +23,10 @@ if(NOT WIN32)
 	target_include_directories(bcrypt PRIVATE "libbcrypt/include/bcrypt")
 endif()
 
+if(CMAKE_CXX_COMPILER_ID MATCHES "Clang|GNU")
```

Comment:
```suggestion
# Need to define this on Clang and GNU for 'strdup' support
if(CMAKE_CXX_COMPILER_ID MATCHES "Clang|GNU")
```
forgot a comment at least acknowledging this, no worries if you dont want to include it.